### ISSUE TITLE
Remove PropTypes

### ISSUE THREAD

**Issue by cmmartin** [mastermoo/react-native-pulse-loader#8]
Latest version of React shipped with react native has removed `PropTypes`, so you'll get 

```
undefined is not an object (evaluating _react.default.PropTypes.number)
```

**Comment by HusseinElGanzory** [comment]
> import PropTypes from "prop-types";

**Comment by ugrdursun** [comment]
Having the same issue, do you know how to fix it ?

> Latest version of React shipped with react native has removed `PropTypes`, so you'll get
> 
> ```
> undefined is not an object (evaluating _react.default.PropTypes.number)
> ```

**Comment by cmmartin** [comment]
@ugrdursun I just copy and pasted the code into my project and deleted all the prop types

**Comment by mastermoo** [comment]
took a while but pr #6 is merged and new version 1.0.3 is published to npm
